frontend/src/download.js

```diff
@@ -29,20 +30,7 @@ $(document).ready(function() {
       // update progress bar
       $('#dl-progress').circleProgress('value', percent);
       $('.percent-number').html(`${Math.floor(percent * 100)}`);
-      if (progress[1] < 1000000) {
-        $('.progress-text').html(
-          `${filename} (${(progress[0] / 1000).toFixed(1)}KB of
-           ${(progress[1] / 1000).toFixed(1)}KB)`
-        );
-      } else if (progress[1] < 1000000000) {
-        $('.progress-text').html(
-          `${filename} (${(progress[0] / 1000000).toFixed(1)}MB of ${(progress[1] / 1000000).toFixed(1)}MB)`
-        );
-      } else {
-        $('.progress-text').html(
-          `${filename} (${(progress[0] / 1000000).toFixed(1)}MB of ${(progress[1] / 1000000000).toFixed(1)}GB)`
-        );
-      }
+      $('.progress-text').text(`${filename} (${bytes(progress[0])} of ${bytes(progress[1])})`);
```

Comment:
I think this may be causing some UI jank as well. You can see in the following UI screenshot that the filesizes are rounded to 2 decimal places. I think in some cases during upload, it's rounding the ".00" to ".0" which causes the UI to slightly shift very briefly.

This _may_ all be academic though since I believe @johngruen also has a PR to tweak the UI, so not sure who's PR will "win".

<img width="580" alt="firefox_send" src="https://user-images.githubusercontent.com/557895/28443865-cb624cb2-6d6d-11e7-9c35-742bdaf98af3.png">